app/models/release.rb

```diff
@@ -44,6 +44,9 @@ def self.find_by_version!(version)
   private
 
   def assign_release_number
+    default_number = 1
```

Comment:
nvm found the test ... I'd still somewhere mention that it's for an external plugin so nobody cleans it up